src/app/components/wrapper-components/rules-configuration/rules-configuration.component.ts

```diff
@@ -94,11 +95,15 @@ export class RulesConfigurationComponent extends BaseWrapperComponent implements
             formControl.disable();
         }
         formControl.valueChanges.subscribe(() => {
-            this.onUpdate();
+            setTimeout(() => this.onUpdate(), 1);
```

Comment:
why? ça semble cacher du bug, au minimum minimum faut expliquer pourquoi c'est nécessaire, au mieux faut rendre ça inutile et enlever le timeout

Reply:
j'ai commenté pourquoi c'est important, et c'est confirmé par UT que c'est nécessaire

Reply:
not update -> not updated
imply -> implies
mais c'est louche cette affaire, valueChanges est appelé alors que la valeur a pas encore changé ?! Ou bien on parle d'une autre valeur ("the value" fait référence à quelle valeur ?)